assets/js/blocks/catalog-sorting/style.scss

```diff
@@ -0,0 +1,7 @@
+.wc-block-catalog-sorting {
```

Comment:
We need to disable the default float style to prevent this block from breaking the layout.

To reproduce:
1. Add `Catalog Sorting` to Product Catalog template.
2. Add `Products (beta)` block.
3. Save change, see error on the front end.

<img width="686" alt="image" src="https://user-images.githubusercontent.com/5423135/212888765-b19c805d-7f84-4a78-bd7e-b12c83245adc.png">

```suggestion
.woocommerce.wc-block-catalog-sorting {
	form {
		float: none;
	}

```